client/Cargo.toml

```diff
@@ -14,27 +14,37 @@ ya-relay-core = "0.4.0"
 ya-packet-trace = "0.1.0"
 
 anyhow = "1.0"
+async-trait = "0.1"
 chrono = "0.4"
 derive_more = "0.99"
+educe = "0.4"
 env_logger = { version = "0.8", default-features = false }
 futures = "0.3"
 humantime = "2.1"
 log = "0.4"
+metrics = "0.21"
+num_cpus = "1.15"
 thiserror = "1.0"
 tokio = { version = "1", features = ["net", "sync", "macros", "time", "rt"] }
 tokio-stream = "0.1.8"
 url = "2.1"
 
+# Necessary for testing purposes
+ya-relay-server = { path="../server"}
```

Comment:
`ya-relay-server` is instantiated in tests. The problem is, that test utils that are doing this, are in `ya-relay-client`.
I would gladly remove this, but it might be difficult

Reply:
Move tests depending on testutils in Client to integration directory